src/main/java/rsb/methods/Combat.java

```diff
@@ -267,14 +268,73 @@ public boolean isAttacking(final RSNPC npc) {
 		return interact != null && interact.equals(npc.getAccessor());
 	}
 
+	public boolean isHPZero(final RSNPC npc) {
```

Comment:
No method comments (Javadocs) for a bunch of new methods.

Reply:
NO, there's ton of WELL DOCUMENTED methods witch don't even work.

Pointless a method should be documented once its validity been well established its performance optimized and its syntax optimized.